backend/project_name/settings/base.py

```diff
@@ -56,7 +56,6 @@ def base_dir_join(*args):
     {
         "BACKEND": "django.template.backends.django.DjangoTemplates",
         "DIRS": [base_dir_join("templates")],
-        "APP_DIRS": True,
```

Comment:
Please test if this breaks apps with templates inside of them. Third-party install-able apps.

Reply:
Like this one: https://github.com/django-crispy-forms/django-crispy-forms

Reply:
I've tested rendering a Crispy Form using settings without `APP_DIRS` and it worked properly, but maybe this specific lib  supports that configuration by default.

Reply:
OK!